pkg/virt-handler/rest/console.go

```diff
@@ -150,7 +161,16 @@ func (t *ConsoleHandler) VNCHandler(request *restful.Request, response *restful.
 	uid := vmi.GetUID()
 	stopChn := newStopChan(uid, t.vncLock, t.vncStopChans)
 	defer deleteStopChan(uid, stopChn, t.vncLock, t.vncStopChans)
-	t.stream(vmi, request, response, unixSocketPath, stopChn)
+	t.stream(vmi, request, response, func() (net.Conn, error) {
+		log.Log.Object(vmi).Infof("Connecting to %s", unixSocketPath)
```

Comment:
We could have one pre-defined callback for unix sockets for vnc, usb and console, only vsock seems to be different atm.

Reply:
Done